config/default/common/config/metadata/stories/hls_intro/step006.md

```diff
@@ -1 +1 @@
-Another trade-off is with how often the satellite instrument collects imagery over a particular part of the Earth. MODIS has a high temporal revisit period therefore there is near daily coverage of the Earth's surface. MSI, aboard Sentinel 2A and 2B, have a 5 day revisit period and OLI, aboard Landsat, has a 16 day revisit period. Combined together by the HLS project, this equates to near global coverage in 2 to 3 days.
+Another trade-off is with how often the satellite instrument collects imagery over a particular part of the Earth. MODIS has a high temporal revisit period therefore there is near daily coverage of the Earth's surface. MSI, aboard Sentinel 2A and 2B, have a 5 day revisit period and OLI, aboard Landsat 8 and 9, has an 8 day revisit period. Combined together by the HLS project, this equates to near global coverage in 2 to 3 days.
```

Comment:
There is some inconsistent grammar ("have" vs "has"). I don't know which way is correct...?

"MSI, aboard Sentinel 2A and 2B, have a..."
"OLI, aboard Landsat 8 and 9, has an 8..."

Reply:
i changed it to have